sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino_source.py

```diff
@@ -97,6 +97,24 @@ def __init__(
         tags: Optional[Dict[str, str]] = None,
         owner: Optional[str] = "",
     ):
+        """
+        Creates a TrinoSource object.
+
+        Args:
+            name (optional): Name for the source. Defaults to the table if not specified.
```

Comment:
What happens if a query is specified instead?

Reply:
in that case, the `name` must be specified

Reply:
Update the doc string to reflect that?